include/autotools.mk

```diff
@@ -1,9 +1,6 @@
+# Copyright 2007-2020 OpenWrt.org
 #
-# Copyright (C) 2007-2012 OpenWrt.org
-#
-# This is free software, licensed under the GNU General Public License v2.
-# See /LICENSE for more information.
-#
+# SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
I think the SPDX license should be in the first line, and the copyright may come afterwards. That's also what the kernel recommendations say:

[...]
The SPDX license identifier in kernel files shall be added at the first possible line in a file which can contain a comment. For the majority or files this is the first line, except for scripts which require the ‘#!PATH_TO_INTERPRETER’ in the first line. For those scripts the SPDX identifier goes into the second line.
[...]

I'm sorry for creating additional work by this ...

Reply:
Yea I remember that too, wonder why the `reuse` tools doesn't follow this idea. I'll patch it